### PR TITLE
Poll questions document and video

### PR DESCRIPTION
**Pull request by MariaCheca**
Where
=====
* **Related Issue:** #1677 
* **Related PR:** https://github.com/consul/consul/pull/1691 

What
====
New feature to attach poll questions:
- 1 video link.
- 1 PDF document.

How
===
- Created migration to add poll questions a new field called `video_url`. The poll question create/edit form has a new input to add the video URL and show it in the question public page.

- Using `Documentable`, added the feature to attach a PDF document to the poll question and make it appear in the question public page, same as the video link.

Screenshots
===========
localhost-3000-admin-questions-new
localhost-3000-questions-63 1
localhost-3000-questions-63
localhost-3000-admin-questions-63 1

Warnings
========
- There are some bugs in the screenshots pending to be fixed.
- **The feature is not completed, adding images to poll questions is still left.**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -662,13 +662,28 @@
...
     t.tsvector "tsv"
+    t.string   "video_url"
   end
 
   add_index "poll_questions", ["author_id"], name: "index_poll_questions_on_author_id", using: :btree
   add_index "poll_questions", ["poll_id"], name: "index_poll_questions_on_poll_id", using: :btree
   add_index "poll_questions", ["proposal_id"], name: "index_poll_questions_on_proposal_id", using: :btree
   add_index "poll_questions", ["tsv"], name: "index_poll_questions_on_tsv", using: :gin
 
+  create_table "poll_recounts", force: :cascade do |t|
```
Comment (bertocq) [inline db/schema.rb]: This table, as well as the tolk ones, should not have been added. Old table schemas on your local database may have the blame, just:

```bash
rake db:reset
rake db:dev_seed
rake db:migrate
```
And you'll see how it goes back to normal (maybe the tipical change for `t.datetime "password_changed_at"` default.. but that must be ignored until we fix it :D )

File: app/views/polls/questions/show.html.erb [hunk 2]
```diff
@@ -55,11 +55,36 @@
   </div>
 </div>
 
+<% if @question.video_url.present? %>
+  <div class='row margin-top poll-question-show'>
```
Comment (decabeza) [inline app/views/polls/questions/show.html.erb]: We always use double quotes `" "` for classes  🤓

File: app/views/polls/questions/show.html.erb [hunk 3]
```diff
@@ -55,11 +55,36 @@
...
+    <div class="small-12 medium-9 column video-link">
```
Comment (decabeza) [inline app/views/polls/questions/show.html.erb]: You must move the `<div class="video-link">` inside the `small-12 medium-9 column` to align correctly this block. 👌 

grid_freak

Comment (MariaCheca) [reply app/views/polls/questions/show.html.erb]: Nice one! I didn't notice 😅

**Pull request merged**